Sources/ObjCSupport/LOKStackLayout.swift

```diff
@@ -50,15 +58,22 @@ extension LOKStackLayoutDistribution {
                       viewReuseId: String? = nil,
                       sublayouts: [LOKLayout]?,
                       configure: ((View) -> Void)? = nil) {
+        self.axis = axis
+        self.spacing = spacing
+        self.distribution = distribution.distribution != nil ? distribution : .fillFlexing
+        self.sublayouts = sublayouts ?? []
+        self.alignment = alignment ?? .fill
+        self.viewClass = viewClass
```

Comment:
Let's make viewClass non-optional here an default to View.self, just like SizeLayout does.